src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -22,10 +22,12 @@ public class AddDependenciesToProjectJson : Task
         // Additional Dependencies to add to the project.json. May Optionally contain a version.
         // Will Override dependencies present in the project if there is a conflict.
         // AdditionalDependencies required metadata:  Name, Version
-
         [Required]
         public ITaskItem[] AdditionalDependencies { get; set; }
 
+        // Framework section which the additional dependencies apply to.  Empty is the default dependencies section.
```

Comment:
@weshaggard , can you provide an example of what your expectation for an empty framework is.  It's not clear to me what the difference you are describing is.

The current change says that, if not Frameworks (TargetGroups) are specified, then add the dependencies to the shared dependencies section...

{
  "Dependencies": {
    "new_dependency": "1.0.0-release-major-minor"
  }
  "Frameworks": {
    // blah
  }
}